examples/SignMultiple.purs

```
@@ -73,19 +76,30 @@ main = launchAff_ $ do
     ubTx1 <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
     ubTx2 <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
 
-    withBalancedAndSignedTxs [ ubTx1, ubTx2 ] $ \txs -> do
+    txIds <- withBalancedAndSignedTxs [ ubTx1, ubTx2 ] $ \txs -> do
       locked <- getLockedInputs
       logInfo' $ "Locked inputs inside bracket (should be nonempty): " <> show
         locked
-      traverse_ submitAndLog txs
+      traverse submitAndLog txs
 
     locked <- getLockedInputs
     logInfo' $ "Locked inputs after bracket (should be empty): " <> show locked
 
+    case txIds of
+      [ txId1, txId2 ] -> do
+        awaitTxConfirmedWithTimeout (wrap 120.0) txId1
+        logInfo' $ "Tx 1 submitted successfully!"
+        awaitTxConfirmedWithTimeout (wrap 120.0) txId2
+        logInfo' $ "Tx 2 submitted successfully!"
+      _ -> logError' "Unexpected error - no transaction IDs"
```

I think we should also throw an error in order for test to really fail.